src/common/components/signInForm/signInForm.component.spec.js

```diff
@@ -95,5 +95,17 @@ describe( 'signInForm', function () {
       expect( $ctrl.errorMessage ).toEqual( 'An error has occurred signing in. Please try again.' );
       expect( $ctrl.isSigningIn ).toEqual( false );
     });
+
+    it( 'removes password from error log', () => {
```

Comment:
@OzzieOrca Can you help me finish this spec?  I'm not sure how to get the response from the rejected deferred.